src/substrait/textplan/tests/CMakeLists.txt

```diff
@@ -18,3 +18,52 @@ add_test_case(
   gmock
   gtest
   gtest_main)
+
+option(SUBSTRAIT_CPP_ROUNDTRIP_TESTING
+       "Enable substrait-cpp textplan roundtrip tests." OFF)
+
+if(${SUBSTRAIT_CPP_ROUNDTRIP_TESTING})
+  add_test_case(
+    round_trip_test
+    SOURCES
+    RoundtripTest.cpp
+    EXTRA_LINK_LIBS
+    substrait_textplan_converter
+    substrait_textplan_loader
+    substrait_common
+    substrait_proto
+    parse_result_matchers
+    protobuf-matchers
+    fmt::fmt-header-only
+    absl::strings
+    gmock
+    gtest
+    gtest_main)
+
+  cmake_path(GET CMAKE_CURRENT_SOURCE_DIR PARENT_PATH TEXTPLAN_SOURCE_DIR)
+
+  add_custom_command(
+    TARGET round_trip_test
+    POST_BUILD
+    COMMAND ${CMAKE_COMMAND} -E echo "Copying unit test data.."
+    COMMAND ${CMAKE_COMMAND} -E make_directory
+            "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/tests/data"
+    COMMAND
+      ${CMAKE_COMMAND} -E copy
+      "${TEXTPLAN_SOURCE_DIR}/converter/data/q6_first_stage.json"
+      "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/tests/data/q6_first_stage.json"
+    COMMAND ${CMAKE_COMMAND} -E copy "${TEXTPLAN_SOURCE_DIR}/data/*.json"
+            "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/tests/data/")
```

Comment:
Is this needed for this test?

Reply:
This copies all of the test data including the test plan used by the converter to be used by the roundtrip test.